data.md

```diff
@@ -725,7 +725,7 @@ Encoding
                     | #rlpEncodeString ( String )       [function]
                     | #rlpEncodeAccount ( Account )     [function]
  // --------------------------------------------------------------
-    rule #rlpEncodeWord(0) => "\x80"
+    rule #rlpEncodeWord(WORD) => "\x80"        requires WORD ==Int 0
     rule #rlpEncodeWord(WORD) => chrChar(WORD) requires WORD >Int 0 andBool WORD <Int 128
```

Comment:
Should definitely be safe to assume this will always be concrete values, matching `0` will be much quicker.